src/components/mentor/mentorshipsList.tsx

```diff
@@ -1,96 +1,165 @@
 import { Avatar, List, ListItem, ListItemText } from '@material-ui/core';
+import AppBar from '@material-ui/core/AppBar';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import createStyles from '@material-ui/core/styles/createStyles';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Typography from '@material-ui/core/Typography';
 import * as React from 'react';
 import { RouteComponentProps, withRouter } from 'react-router';
 import styled from 'styled-components';
-import { ACTIVE, Const } from '../../Const';
+import { ACTIVE, PENDING } from '../../Const';
 import { Api } from '../../services/api';
 import { SessionService } from '../../services/sessionService';
 import { IMentorship } from '../shared/models/mentorship';
 import { NoContent } from '../shared/noConent';
 
-interface IMentorshipsListProps {
+interface IMentorshipsListRouterParams {
   id: string;
 }
 
 interface IMentorshipsListState {
-  mentorships: IMentorship[];
+  activeMentorships: IMentorship[];
+  pendingMentorships: IMentorship[];
+  selectedTab: number;
 }
 
+type fProps = RouteComponentProps<IMentorshipsListRouterParams> & WithStyles<typeof styles>;
+
 const MentorshipsListWrapper = styled.div`
   height: 100%;
 `;
 
-class MentorshipsList extends React.Component<RouteComponentProps<IMentorshipsListProps>, IMentorshipsListState> {
-  constructor(props: RouteComponentProps<IMentorshipsListProps>) {
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      height: '100%',
+      flexGrow: 1,
+      backgroundColor: theme.palette.background.paper
+    }
+  });
+
+const TabContainer: React.SFC<{ children: React.ReactNode }> = props => {
```

Comment:
Its okay to leave out the type argument for `StatelessComponent`.
```suggestion
const TabContainer: React.SFC = props => {
```